src/main/java/domain/RacingGameResult.java

```diff
@@ -21,4 +21,41 @@ public List<RoundResult> getAllRoundResults() {
     public RoundResult getLastRoundResult() {
         return roundResult.get(roundResult.size() - 1);
     }
+
+    public List<String> whoAreWinners() {
+        RoundResult lastRoundResult = getLastRoundResult();
+        List<RacingCarCurrentStatus> raceResult = lastRoundResult.getRaceProgress();
+
+        Integer maxPosition = findMaxPosition(raceResult);
+        return findWinners(raceResult, maxPosition);
+    }
+
+    private List<String> findWinners(List<RacingCarCurrentStatus> raceResult, Integer maxPosition) {
+        List<String> winners = new ArrayList<>();
+        for (RacingCarCurrentStatus car : raceResult) {
+            addIfWinner(car, maxPosition, winners);
+        }
+        return winners;
+    }
+
+    private void addIfWinner(RacingCarCurrentStatus car, Integer maxPosition, List<String> winners) {
+        if (maxPosition.equals(car.whereIsThisCarNow())) {
+            winners.add(car.whatNameIsThisCar());
+        }
+    }
+
+    private Integer findMaxPosition(List<RacingCarCurrentStatus> raceResult) {
+        int maxPosition = 0;
+        for (RacingCarCurrentStatus car : raceResult) {
+            maxPosition = updateMaxPosition(car, maxPosition);
+        }
+        return maxPosition;
+    }
+
+    private Integer updateMaxPosition(RacingCarCurrentStatus car, int maxPosition) {
+        if (car.whereIsThisCarNow() > maxPosition) {
```

Comment:
```suggestion
        if (car.forwardThan(maxPosition)) {
```

상태를 가진 객체에게 메시지를 통해 물어보면 어떨까요?

Reply:
**책임과 데이터를 동일 객체로 묶는 객체 지향의 핵심 원칙**을 준수하는 코드로 변경하는 것으로 이해해도 될까요?